common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -36,6 +36,11 @@ object Messages {
     deviceId: Option[DeviceId],
     deviceType: DeviceType) extends BusMessage
 
+  final case class DeviceCredentialsSet(
```

Comment:
This is just the public data (ClientId or Certificate) so that should be alright, after talking with Jerry I think I will rename it to Certificate (unless you have a better name for the public part).